05_stringer/patrickmarabeas/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+type IPAddr [4]byte
```

Comment:
It is good to get in the habit of documenting exported types and functions

```go
// IPAddr holds an IP Address by storing 4 bytes.
type IPAddr [4]byte
```

Or something similar..

Reply:
Added doc strings.